### PR TITLE
Add Tibetan Amdo dialect xbmu_amdo31 in egs

### PR DESCRIPTION
**Pull request by sendream**

### CODE REVIEW

File: egs/xbmu_amdo31/ASR/RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+```bash
+cd egs/xbmu_amdo31/ASR
+./prepare.sh
+
+export CUDA_VISIBLE_DEVICES="0"
+./pruned_transducer_stateless5/train.py 
+```
+
+**Caution**: It uses `--context-size=1`.
+
```
Comment (csukuangfj) [inline egs/xbmu_amdo31/ASR/RESULTS.md:27]: Could you also upload the pre-trained model to hugging face?

Comment (sendream) [reply egs/xbmu_amdo31/ASR/RESULTS.md:27]: > Could you also upload the pre-trained modelok to hugging face?
ok

**Comment by sendream** [comment]
The task has an active leaderboard which can be found at https://github.com/sendream/xbmu_amdo31/blob/main/README.md#leaderboard and ranks models based on their WER.

**Comment by csukuangfj** [comment]
Thanks for your contribution!

**Review by csukuangfj — APPROVED** [review]

**Comment by csukuangfj** [comment]
By the way, you can use the pre-trained models in your browser

https://huggingface.co/spaces/k2-fsa/automatic-speech-recognition

image

**Pull request merged**